packages/graphql/src/classes/Neo4jGraphQL.ts

```diff
@@ -128,6 +128,50 @@ class Neo4jGraphQL {
         await assertIndexesAndConstraints({ driver, driverConfig, nodes: this.nodes, options: input.options });
     }
 
+    private wrapResolvers(resolvers: IResolvers, { schema }: { schema: GraphQLSchema }) {
+        const wrapResolverArgs = {
+            driver: this.driver,
+            config: this.config,
+            nodes: this.nodes,
+            relationships: this.relationships,
+            schema,
+        };
+
+        const resolversComposition = {
+            "Query.*": [wrapResolver(wrapResolverArgs)],
+            "Mutation.*": [wrapResolver(wrapResolverArgs)],
+        };
+
+        // this.schemaDefinition.resolvers can either be undefined, IResolvers or IResolvers[]
+        let allResolvers = [resolvers];
+        if (this.schemaDefinition.resolvers) {
+            if (Array.isArray(this.schemaDefinition.resolvers)) {
+                allResolvers = [...allResolvers, ...this.schemaDefinition.resolvers];
+            } else {
+                allResolvers.push(this.schemaDefinition.resolvers);
+            }
+        }
+
+        // Merge generated and custom resolvers
+        const mergedResolvers = mergeResolvers(allResolvers);
+
+        return composeResolvers(mergedResolvers, resolversComposition);
+    }
+
+    private addWrappedResolversToSchema(resolverlessSchema: GraphQLSchema, resolvers: IResolvers): GraphQLSchema {
+        const schema = addResolversToSchema(resolverlessSchema, resolvers);
+
+        // Assign a default field resolver to account for aliasing of fields
+        forEachField(schema, (field) => {
+            if (!field.resolve) {
+                // eslint-disable-next-line no-param-reassign
+                field.resolve = defaultFieldResolver;
+            }
+        });
```

Comment:
**suggestion (non-blocking)**: I know it may be pushing a bit, but it could be sooo nice
```suggestion
    return addDefaultResolvers(schema)
```